driver-core/src/main/com/mongodb/connection/CommandMessage.java

```diff
@@ -113,6 +113,7 @@ ReadPreference getReadPreference() {
     protected EncodingMetadata encodeMessageBodyWithMetadata(final BsonOutput bsonOutput, final SessionContext sessionContext) {
         int commandStartPosition;
         if (useOpMsg()) {
+            int startPosition = bsonOutput.getPosition() - MESSAGE_PROLOGUE_LENGTH;
```

Comment:
Should you call this variable messageStartPosition for clarity?

Reply:
Done